web/themes/custom/atrium/templates/page/page.html.twig

```diff
@@ -62,14 +62,16 @@
     </div>
   {% endif %}
 
-  {% set nodes_with_hero = [
+  {% set entities_with_hero = [
     'srl_howto_instruction',
     'srl_howto',
     'srl_onramp',
     'srl_wayfinder',
+    'jcc_form'
   ] %}
 
-  {% if page.title|render and node.bundle() not in nodes_with_hero %}
+  {% if page.title|render and entity_bundle not in entities_with_hero %}
+
     <div class="page__title">
       {{ page.title }}
     </div>
```

Comment:
This custom work allow for the hero to be set on the media or node template instead of in the page template. If we did this globally, then non-entity pages like the user login form wouldn't have a title.